src/macros.jl

```diff
@@ -436,6 +436,10 @@ end
 
 # three-argument build_constraint is used for two-sided constraints.
 build_constraint(_error::Function, func::AbstractJuMPScalar, lb::Real, ub::Real) = build_constraint(_error, func, MOI.Interval(lb, ub))
+# To create `MOI.SingleVariable`-in-`MOI.Interval`, use
+# `@constraint(model, var in MOI.Interval(lb, ub)`.
+# See https://github.com/JuliaOpt/JuMP.jl/issues/1586
```

Comment:
Nit: We shouldn't be littering the code with issue links. In this case the conclusion of the issue can be summarized concisely (we do this for consistency with how one-sided (in)equality constraints are parsed).